lib/util/backend_data.dart

```diff
@@ -0,0 +1,70 @@
+/// enumeration for gender-variable.
+enum Gender {
+  /// element for denoting the title of men
+  male,
+
+  /// element for denoting the title of women
+  female,
+}
+
+// TODO: is there a better approach?
+
+/// class for personal data variables.
+class PersonalData {
+  /// stores the first name of the patient;
+  static String firstName = '';
+
+  /// stores the last name of the patient.
+  static String familyName = '';
+
+  /// stores the email-address of the patient.
+  static String email = '';
+
+  /// stores the phone number of the patient.
+  static String number = '';
+
+  /// stores the address of the patient.
+  static String address = '';
+
+  /// contains the Gender of the patient.
+  static Gender gender = Gender.female;
+}
+
+/// class for parameter variables.
+class Parameters {
+  /// contains patient's weight.
+  static bool weightBMIEnabled = false;
+
+  /// contains patient's heart frequency.
+  static bool heartFrequencyEnabled = false;
+
+  /// contains patient's blood pressure.
+  static bool bloodPressureEnabled = false;
+
+  /// contains patient's blood sugar level.
+  static bool bloodSugarLevelsEnabled = false;
+
+  /// contains patient's walk distance.
+  static bool walkDistanceEnabled = false;
+
+  /// contains patient's sleep duration.
+  static bool sleepDurationEnabled = false;
+
+  /// contains patient's sleep quality.
+  static bool sleepQualityEnabled = false;
+
+  /// contains patient's pain.
+  static bool painEnabled = false;
+
+  /// contains patient's PHQ-4.
+  static bool phq4Enabled = false;
+
+  /// contains patient's medication.
+  static bool medicationEnabled = false;
+
+  /// contains patient's therapy.
+  static bool therapyEnabled = false;
+
+  /// contains patient's doctor's visit.
+  static bool doctorsVisitEnabled = false;
+}
```

Comment:
As I understand, you are having here a class that only holds static values so you can use them inside the createPatient() method inside the Backend class.

The proper way of doing this would be to create a model and then using it, creating an object based on the model. For saving it the createPatient() method should take the Patient as a parameter.
If you have any difficulties with my proposal, let me know.